tests/functional/test_contract_event.py

```diff
@@ -277,6 +277,7 @@ def test_contract_two_events_with_same_name(
 def test_contract_decode_logs_no_abi(owner, contract_instance):
     receipt = contract_instance.setNumber(1, sender=owner)
     events = list(receipt.decode_logs())  # no abi
+    print(contract_instance.NumberChange(1))
```

Comment:
No print statements if not necessary

Reply:
Yeah this was just my attempt to test out the `__call__` implementation, I only pushed to show what I was trying (and failing)